### PR TITLE
[timeout@narutrey] Applet disabled since Cinnamon 5.4 because session loss risk

### LINKED ISSUE #5162: Time Out (timeout@narutrey): Not working, Risk of loosing sessionm, Please unpublish.
### Applet version/Build date

2023-05-18 07:42:34

### Cinnamon version

5.6.8

### Distribution

Mint 21

### Graphics hardware *and* driver used

NVIDIA Corporation GA104M [GeForce RTX 3070 Mobile / Max-Q]

### Applet name and maintainer

timeout@narutrey @narutrey

### What happened?

I activated the timer. After the set time, it blended my desktop as expected, told me to take a break as expected. I hit `postpone`, nothing happened to the timer. even waiting for a looong time didn't make the overlay go away.
I tried everything I could, including [Ctrl][Alt][Del] and [Ctrl][Alt][F5] and trying to kill the process via `top` & `kill -9`, but no success. I had no chance to regain my machine but `sudo shutdown now`.

**I expect this applet to be unpublished until the danger for users to lose their work is banned. The applet supersedes most critical key combinations and is thus to be declared unsafe and dangerous.**

### Other information

_No response_

### PR DESCRIPTION
**Pull request by claudiux**
Fixes #5162 (temporarily).

### CODE REVIEW

File: timeout@narutrey/info.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
-    "author": "narutrey",
+    "author": "",
+    "original-authors": [
+        "narutrey"
+    ],
```
Comment (rcalixte) [inline timeout@narutrey/info.json:5]: This should be:

```json
  "author": "none",
  "original_author": "narutrey"
```

File: timeout@narutrey/files/timeout@narutrey/5.4/applet.js [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+        if (isShowHours) {
+            label = h + ':' + m;
+        } else {
+            m = parseInt(m, 10); // Drop leading zero
+            if (seconds % 60 > 0) ++m;
+            label = '' + m;
+        }
+    }
+    return label;
+}
```
Comment (rcalixte) [inline timeout@narutrey/files/timeout@narutrey/5.4/applet.js:395]: Take a look [here](https://github.com/linuxmint/cinnamon-spices-extensions/tree/master/mnemonic-5-4-window-menu%40mtwebster/files/mnemonic-5-4-window-menu%40mtwebster) for how to properly disable this past a certain point. You would need to create a 5.6 directory (assuming you still want 5.4 supported) with null functions. If you don't want this enabled for 5.4, then move the full code to 5.2 and the null functions to a 5.4 subdirectory.

**Review by rcalixte — CHANGES_REQUESTED** [review]

File: timeout@narutrey/info.json (hunk 1 above)
Comment (claudiux) [reply timeout@narutrey/info.json:5]: Ok but many applets have the  "original-authors" field in their... metadata.json oops.

Comment (claudiux) [reply timeout@narutrey/info.json]: Changes made.

Comment (rcalixte) [reply timeout@narutrey/info.json]: The author key should be "none" instead of an empty string.

Comment (claudiux) [reply timeout@narutrey/info.json]: Done

**Pull request closed without merging**